docs/docs/overview/architecture.md

```diff
@@ -11,13 +11,13 @@ Let's deep dive into the building blocks of Novu's OCL approach.
 
 ## Templates
 
-Templates are the blueprints for all notifications in Novu. They provide the base configurations for each message. A message is tied to a specific channel, for which a content template is provided, code rules and filters, priorities, and other metadata that will affect the delivery of a specific message.
+Templates are the blueprints for all the notifications in Novu. They provide base configurations for each message. The message is tied to a specific channel for which a content template, code rules and filters, priorities, and other metadata that will affect the delivery of a specific message is provided.
 
 ## Environments
 
-This is the context in which all of your subscribers and templates exist. This will usually map to your own environments, so any new changes you are making will first be in the Development environment and once tested, you can use our merging changes tool to promote them to production.
+This is the context in which all of your subscribers and templates exist. This will usually map to your own environments, so any new changes you are making will be  firstly included in the Development environment and once they are tested, you can use our merging changes tool to add them to production.
```

Comment:
```suggestion
This is the context in which all of your subscribers and templates exist. This will usually map to your own environments, so any new changes you are making will be  firstly included in the Development environment and once they are tested, you can use our merging changes tool to promote them to production environment.
```